installers/ansible/roles/pgo-operator/files/pgo-configs/cluster-deployment.json

```diff
@@ -145,6 +145,18 @@
                     }, {
                         "name": "PGHOST",
                         "value": "/tmp"
+                    }, 
+                    {
+                      "name": "LD_PRELOAD",
+                      "value": "/usr/lib64/libnss_wrapper.so"
+                    },
+                    {
+                      "name": "NSS_WRAPPER_PASSWD",
+                      "value": "/tmp/nss_wrapper/postgres-ha/passwd"
```

Comment:
Noting that this needs to be reviewed in the context of https://github.com/CrunchyData/crunchy-containers/pull/1337 based on where some of those settings are being stored based on that name (e.g. `postgres-ha`).

Reply:
Thanks, great catch - think the answer here will be to keep the directory the same across all PG containers, whether HA, GIS, etc.  I'll adjust accordingly.

Reply:
Updated the path to `/tmp/nss_wrapper/postgres`, as will now be utilized for all variants of the PG container (HA, PostGIS, etc.).